### DISCUSSION TITLE
There is space.

### DISCUSSION THREAD

**Discussion by jukqaz** [MobileReality/react-native-select-pro#127] [Q&A] [answered]
## Image
![Screenshot 2022-11-16 at 2 08 13 PM](https://user-images.githubusercontent.com/6260165/202089050-a0f3a856-bd5c-4d48-a075-7ccfcf4ab792.png)

## Code
```ts
{devices && (
  <SelectProvider>
    <PaddingContainer style={{ width: '100%' }}>
      <Select
        animated
        placeholderText={'Select device'}
        onSelect={(_, index) => selectDevices(index)}
        options={devices.map((device) => ({
          label: device.deviceId,
          value: device.deviceNickname,
        }))}
      />
    </PaddingContainer>
  </SelectProvider>
)}
```

## Explain
As in the code shown above, I used only the basic functions, but there is a space as above in Android and iOS. 
What should I fix?

**Comment by jukqaz** [comment]
When ```optionsListStyle={{ marginTop: -97 }}``` is applied, the space disappears, but it seems to be an incorrect method.

**Comment by kkaplinski** [comment, marked as answer]
Hi @jukqaz. Please try to wrap your whole app with ` <SelectProvider>` in root component: https://mobilereality.github.io/react-native-select-pro/docs/api/select-provider.

If it doesn't help please provide us with some reproducible expo snack or minimum repo.

**Reply by jukqaz** [reply]
Thank you for your support!
